### ISSUE TITLE
Pivot does not support overflow per WCAG2.1 Reflow requirement

### ISSUE THREAD

**Issue by ayuspark** [OfficeDev/office-ui-fabric-react#11863]
### Environment Information

- **Package version(s)**: 7.88
- **Browser and OS versions**: Chrome 9.0.3945.130, and other browsers (FF/Edge) as well. 
OS:  Windows 10 enterprise, Version: 10.0.18363 Build 18363

### Describe the issue:

Per [WCAG2.1 Reflow requirement](https://www.w3.org/TR/WCAG21/#reflow), we need to have the UI responsive down to 1280x1024 at 400% zoom (and everything in between); all content should be accessible with no bi-directional scrolling (except content which require two-dimensional layout are images, maps, diagrams, video, games, presentations, data tables, and interfaces where it is necessary to keep toolbars in view while manipulating content).

Current `<Pivot />` component does not hide overflow `PivotItem`s when zoomed in at 400%. 

#### Actual behavior:

`PivotItem` not collapsed.

#### Expected behavior:

PivotItem collapsed like in `OverflowSet` items.

**Comment by xugao** [comment]
@ayuspark - thanks for reporting. currently `Pivot` control is not designed to be responsive. I will check with design on what the expected experience should be. However, i don't think we can get this implemented in the short-term

**Comment by ayuspark** [comment]
> @ayuspark - thanks for reporting. currently `Pivot` control is not designed to be responsive. I will check with design on what the expected experience should be. However, i don't think we can get this implemented in the short-term

@xugao thank you for getting back so quickly!
I'm working on a Microsoft product and we heavily rely on Fabric UI and BAF. I can also see that in many other products under Dynamics there're customized component written to bypass the <Pivot /> reflow issue. Maybe it's worth investigating on it :)

**Comment by xugao** [comment]
@ayuspark - definitely! In fact, this is something we tried to tackle before. see #4066. We will continue to follow up there and close this one as duplicate
